pkg/client/interfaces.go

```diff
@@ -89,7 +89,7 @@ type Client interface {
 }
 
 // IndexerFunc knows how to take an object and turn it into a series
-// of (non-namespaced) keys for that object.
+// of non-namespaced keys. Namespaced objects are automatically wrapped.
```

Comment:
maybe make this `Namespaced objects are automatically given namespaced and non-spaced variants, so keys do not need to include namespace`.

Reply:
Fixed!